lib/bolt/cli.rb

```diff
@@ -129,6 +130,10 @@ def initialize(options)
                         '* port defaults to `5985` or `5986` for WinRM, based on the --[no-]ssl setting') do |nodes|
           @options[:nodes] << get_arg_input(nodes)
         end.extend(SwitchHider)
+        @query = define('-q', '--query QUERY',
```

Comment:
Having query be specific to PuppetDB might bite us later. But it certainly seems like the best option for usability right now.

Reply:
I agree with both of those points. I think I would prefer to keep it straightforward now and complicate it later if necessary.